src/main/scala/util/Vector3.scala

```diff
@@ -0,0 +1,10 @@
+package util
+
+/**
+  * Class representing a location on a 3D-axis system
+  * @param X The value on the X-axis
+  * @param Y The value on the Y-axis
+  * @param Z The value on the Z-axis
+  */
+case class Vector3(X: Double, Y: Double, Z: Double) {
```

Comment:
Isn't there a library for a class like this? It saves a lot of work/time when vector calculations a necessary in the future.

Reply:
Is this fixed? I can't see any change or discussion but you did approve the PR.